class.jetpack-gutenberg.php

```diff
@@ -291,7 +307,7 @@ public static function get_blocks_directory() {
 	 * @return bool True if the file is found.
 	 */
 	public static function preset_exists( $preset ) {
-		return file_exists( JETPACK__PLUGIN_DIR . self::get_blocks_directory() . $preset . '.json' );
+		return file_exists( JETPACK__PLUGIN_DIR . self::get_blocks_source_directory() . 'setup/' . $preset . '.json' );
```

Comment:
Yeah, not super elegant :roll_eyes: Not really sure if there's any way around this.

Reply:
Do we need this method at all? Can we remove `preset_exists` and `get_preset` in #11747?